app/views/shared/_google_analytics.html.erb

```diff
@@ -0,0 +1,9 @@
+<script>
```

Comment:
should we put an `if` around this so it only runs in prod? 

Also, staging and prod are both `production` so maybe we could just set the `property Id` via an env var so we aren't accidentally tracking metrics for both under the same account?